### PR TITLE
feat(BigQuery): add saveEmailsData to big query http client

### PR DESCRIPTION
**Pull request by kapoc3**
Method for save data emails added on http bigquery client.

### CODE REVIEW

File: src/services/big-query-client.ts [hunk 1]
```diff
@@ -1,12 +1,15 @@
-import { ResultWithoutExpectedErrors } from '../doppler-types';
 import { HttpBaseClient } from './http-base-client';
+import { ResultWithoutExpectedErrors, EmptyResultWithoutExpectedErrors } from '../doppler-types';
 
+export type SaveEmailsResult = EmptyResultWithoutExpectedErrors;
 export interface EmailList {
+  idUser?: String;
```
Comment (andresmoschini) [inline src/services/big-query-client.ts:6]: ¿Por qué agregás este campo?

Comment (kapoc3) [reply src/services/big-query-client.ts:6]: olvide removerlo del cliente, solucionado!!

File: src/services/big-query-client.ts [hunk 2]
```diff
@@ -35,4 +37,18 @@ export class HttpBigQueryClient extends HttpBaseClient implements BigQueryClient
       return { success: false, error: error };
     }
   }
+
+  public async saveEmailsData(payload: EmailList): Promise<SaveEmailsResult> {
+    const { jwtToken, email } = this.getApiConnectionData(this.clientName);
+    const response = await this.axios.post(`/big-query/${email}/save-emails`, {
```
Comment (andresmoschini) [inline src/services/big-query-client.ts:43]: In my opinion, it should be a `PUT` against the same URL:

```suggestion
    const response = await this.axios.post(`/big-query/${email}/allowed-emails`, {
```

In my opinion, it is the same resource that you obtain with the `GET`, also, it is a kind of update, an idempotent operation, so `PUT` fits much better.

Comment (kapoc3) [reply src/services/big-query-client.ts:43]: suggest accepted and applied!!!

**Comment by cbernat** [comment]
@kapoc3 it would be nice making use of this method in what you have already in the view right?

**Comment by kapoc3** [comment]
> 
> 
> @kapoc3 it would be nice making use of this method in what you have already in the view right?

@cbernat when the junior's component be integrate with the template, i can generate a new pr with the integration of button save

**Comment by cbernat** [comment]
> > @kapoc3 it would be nice making use of this method in what you have already in the view right?
> 
> @cbernat when the junior's component be integrate with the template, i can generate a new pr with the integration of button save

But you have a small implementation of your own that can be tested right?

**Comment by cbernat** [comment]
Can you add all pages related to big query here? https://app.fromdoppler.com/new-features we have this page for pending development, to access them easily in the meantime.

**Comment by cbernat** [comment]
> > > @kapoc3 it would be nice making use of this method in what you have already in the view right?
> > 
> > 
> > @cbernat when the junior's component be integrate with the template, i can generate a new pr with the integration of button save
> 
> But you have a small implementation of your own that can be tested right?

I've seen that big query page has not submit button yet 🤦 So ignore this comment.

**Comment by cbernat** [comment]
👍 🚀

**Pull request merged**